### ISSUE TITLE
Make mounted snapshots read-only.

### ISSUE THREAD

**Issue by stevenjswanson** [NVSL/linux-nova#48]
Currently, if you mount a snapshot and modify it, the changes appear in the current FS image.

Snapshot mounts should be read-only.

**Comment by Andiry** [comment]
Correct. File system read-only is an important feature that currently lacked.
